applications/CMakeLists.txt

```diff
@@ -22,7 +22,16 @@ target_compile_definitions(segyio-cath PRIVATE
     -Dsegyio_MINOR=${segyio_MINOR}
 )
 
+add_executable(segyio-catb segyio-catb.c)
+target_link_libraries(segyio-catb segyio)
+target_compile_definitions(segyio-catb PRIVATE
+    -Dsegyio_MAJOR=${segyio_MAJOR}
+    -Dsegyio_MINOR=${segyio_MINOR}
+)
 install(TARGETS segyinfo DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
 install(TARGETS segyinspect DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
 
 install(TARGETS segyio-cath DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(TARGETS segyio-catb DESTINATION ${CMAKE_INSTALL_BINDIR})
+
+add_python_test(python.segyb        test/test_segyb.py)
```

Comment:
Noticed this was the case in all the files in the PR, so I've added newline to them all